cabotage/utils/release_build_context.py

```diff
@@ -1,11 +1,19 @@
 import kubernetes
 
+# NOTE: Because we run the entrypoint as `nobody` with a `/nonexistent`
```

Comment:
nit: move this comment into `RELEASE_DOCKERFILE_TEMPLATE` so it is visible to end-users in case they go digging. since the "this is what we built for you" docker file is shown in the UI